src/transformers/models/timm_backbone/modeling_timm_backbone.py

```diff
@@ -72,6 +72,13 @@ def __init__(self, config, **kwargs):
             out_indices=out_indices,
             **kwargs,
         )
+
+        # Converts all `BatchNorm2d` and `SyncBatchNorm` or `BatchNormAct2d` and `SyncBatchNormAct2d` layers of provided module into `FrozenBatchNorm2d` or `FrozenBatchNormAct2d` respectively
+        if getattr(config, "freeze_batch_norm_2d", False):
+            self.freeze_batch_norm_2d()
+        else:
+            self.unfreeze_batch_norm_2d()
```

Comment:
I'm not sure that we want to do `unfreeze_batch_norm_2d` automatically if the argument isn't in the config. If I loaded in a saved backbone, which previously did not have this argument in its config but did have frozen batch norm layers - this will now unfreeze them silently. Moreover, it's not obvious that `freeze_batch_norm_2d=False` unfreezes, it can equally be read as just not freezing. 

Instead, I would only apply if the argument is positive, and let the user unfreeze directly with `model.unfreeze_batch_norm_2d` if they need.  

```suggestion        
        # Converts all `BatchNorm2d` and `SyncBatchNorm` or `BatchNormAct2d` and `SyncBatchNormAct2d` layers of provided module into `FrozenBatchNorm2d` or `FrozenBatchNormAct2d` respectively
        if getattr(config, "freeze_batch_norm_2d", False):
            self.freeze_batch_norm_2d()
```

Reply:
makes total sense :)